Add `DROP DATABASE name CASCADE ALIASES`

Hunterness opened the pull request:
to drop any aliases blocking the deletion of the database

Documenting https://github.com/neo-technology/neo4j/pull/26465, ~~needs to wait until that has been merged.~~

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -3,7 +3,7 @@
 [[composite-databases-delete]]
 = Delete composite databases
 
-You can delete composite databases using either the command `DROP COMPOSITE DATABASE name` or the more general one  `DROP DATABASE name`.
```
Not sure this should be 'You'. Probably this paragraph needs some editing.

Hunterness replied:
this is the current version (I just removed a space where there were 2 between the words), I'll leave it to the docs team to decide on the `you`

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
+[[composite-databases-delete-with-aliases]]
+== Delete a composite database with constituent database aliases
+
+In general, deleting a composite database that has constituent database aliases will fail due to those database aliases.
```
```suggestion
In general, deleting a composite database that has constituent database aliases will fail as these aliases need to be dropped first.
```

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
+== Delete a composite database with constituent database aliases
+
+In general, deleting a composite database that has constituent database aliases will fail due to those database aliases.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
```
```suggestion
This applies to both the local and remote constituent database aliases belonging to the specified composite database.
```

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
+In general, deleting a composite database that has constituent database aliases will fail due to those database aliases.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+By appending `CASCADE ALIASES` to the command `DROP COMPOSITE DATABASE`, those constituent database aliases can be dropped together with the composite database.
```
```suggestion
By appending `CASCADE ALIASES` to the command `DROP COMPOSITE DATABASE`, those constituent database aliases can be dropped alongside the composite database.
```

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,34 @@ However, keep in mind that the first command will fail if the target database is
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+By appending `CASCADE ALIASES` to the command `DROP COMPOSITE DATABASE`, those constituent database aliases can be dropped together with the composite database.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
```
Do we need to repeat this here?

mnd999 replied:
Yeah, maybe we should mention that. I don't think you can overdocument cascading deletes.

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
+DROP COMPOSITE DATABASE movies CASCADE ALIASES
+----
+
+This behaviour would be the same for the more general one `DROP DATABASE name` as well when using it to drop composite databases.
```
```suggestion
This behaviour would be the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
```
Also not sure about the British spelling behaviour, maybe it should be the US behavior

Hunterness replied:
it probably should yes
```suggestion
This behavior would be the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
```

mnd999 commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
+
+[NOTE]
+====
+The `CASCADE ALIASES` option will drop the same aliases that would be given in the error message from trying to drop the database using the `RESTRICT` mode instead.
```
Do we need this note? It seems like it could easily get broken by changing the error message and it kinda restates the above.

mnd999 replied:
I'm not entirely convinced by the comment, but if we want to address it, I don't think this is a good way because running the command to get the error message will actually drop the database if there are no aliases.

Perhaps something like: 

The aliases that will be dropped can be shown with `SHOW DATABASE db YIELD name, constituents` . And the same for aliases for the other page.

Hunterness replied:
I mean you were kinda after trying to drop the database regardless 🤷 but I'll update to something along those lines tomorrow 👍

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,46 @@ An example could look like this:
+[[delete-databases-with-aliases]]
+=== Delete a database with local database aliases targeting it
+
+In general, deleting a database that is the target of any local database aliases will fail due to those database aliases.
```
```suggestion
In general, deleting a database that is the target of any local database aliases will fail due as these aliases need to be dropped first.
```

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,46 @@ An example could look like this:
+=== Delete a database with local database aliases targeting it
+
+In general, deleting a database that is the target of any local database aliases will fail due to those database aliases.
+Any remote database aliases targeting the database does not affect the deletion of the database.
```
```suggestion
Remote database aliases targeting the database do not affect the deletion of the database.
```

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,46 @@ An example could look like this:
+In general, deleting a database that is the target of any local database aliases will fail due to those database aliases.
+Any remote database aliases targeting the database does not affect the deletion of the database.
+
+By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped together with the database.
```
```suggestion
By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped alongside the database.
```

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,46 @@ An example could look like this:
+Any remote database aliases targeting the database does not affect the deletion of the database.
+
+By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped together with the database.
+Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behaviour as if you create them targeting a non-existing database).
```
```suggestion
Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behaviour as if they are created targeting a non-existing database).
```
Again US / UK spelling of behaviour, docs team probably can confirm.

Hunterness replied:
```suggestion
Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behavior as if they are created targeting a non-existing database).
```

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,44 @@ An example could look like this:
+Any remote database aliases targeting the database does not affect the deletion of the database.
+
+By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped together with the database.
+Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behaviour as if you create them targeting a non-existing database).
```
Doesn't do any harm, although I don't think I would assume it would.

mnd999 commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,46 @@ An example could look like this:
+
+[NOTE]
+====
+The `CASCADE ALIASES` option will drop the same aliases that would be given in the error message from trying to drop the database using the `RESTRICT` mode instead.
```
As above, I don't think we should mention the error message.

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

Hunterness commented:
@renetapopova I guess you are busy with the linked users PR, but I'm just looking at the status of my open PRs and want to know that this one hasn't been forgotten about :)

renetapopova commented:
> @renetapopova I guess you are busy with the linked users PR, but I'm just looking at the status of my open PRs and want to know that this one hasn't been forgotten about :)

Actually, I added some comments, but I wanted to look at it again. I'll submit them now and look at the PR again later today.

renetapopova reviewed (changes requested):
I added some suggestions.

Hunterness commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,47 @@ An example could look like this:
+
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege (see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges]).
+
+Given database and database alias:
```
wouldn't this and the next bit make them into two separate examples? this is more set-up for the drop part and feels odd to have as an example on its own here?

renetapopova replied:
Actually, I would suggest to be one example within an example framework. But I need to build it locally to see how it will look. I find it difficult to see the result from just commenting.

renetapopova replied:
I'll suggest something.

renetapopova replied:
We can have a similar example and wording for the other example. But I think we should see if we can reorganize that page as well, as at the moment, it discusses two topics—one is in the intro (_You can delete composite databases using either the command DROP COMPOSITE DATABASE name or the more general one DROP DATABASE name_), and the other is the new section, which is another way of dropping a composite database.

Hunterness commented:
> Actually, I added some comments, but I wanted to look at it again. I'll submit them now and look at the PR again later today.

I've updated some of them so hopefully that helps when looking at it again

renetapopova commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,49 @@ An example could look like this:
+----
+DROP DATABASE movies CASCADE ALIASES
+----
+
```
```suggestion
====
```

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

Hunterness commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -3,12 +3,63 @@
+
+The `CASCADE ALIASES` option is useful when you want to delete a composite database and its constituent database aliases in one step.
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege.
+For more information, see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges].
```
```suggestion
For more information about this privilege, see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges].
```
since that page don't really talk about drop database cascade

Hunterness commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -3,12 +3,63 @@
+
+This behavior is the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
+
+The option `RESTRICT` explicitly requests the default behavior of the command.
```
since this section don't have a syntax description nor mention RESTRICT before this point, this sentence sticks out weirdly

renetapopova replied:
Yes, I had this question in my mind: What is the default behavior of the command?

Hunterness replied:
to throw an error when there exists aliases (for composite its constituent aliases and for standard it's local ones targeting it), so the behaviour if you don't write `CASCADE ALIASES`

renetapopova replied:
Ah ok. And when or why would one use it?

Hunterness replied:
I don't really see people using it as it means the same as leaving it out, but it is there for consistency. We do allow you to specify the default behaviours of DUMP DATA/NOWAIT so why wouldn't we allow you to specify RESTRICT

Hunterness replied:
I think leaving it out on the composite page is fine

Hunterness commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,54 @@ An example could look like this:
+There are two ways of dropping a database that is the target of local database aliases:
+
+* Drop the local database aliases first, then use `DROP DATABASE name` to drop the database.
+Remote database aliases targeting the database do not affect the deletion of the database.
```
```suggestion
Remote database aliases targeting the database do not affect the deletion of the database and therefore dos not need to be dropped beforehand.
```

renetapopova reviewed (approved):
I think it looks good now. @Hunterness, could you please take a look? Thanks

renetapopova commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,54 @@ An example could look like this:
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege.
+For more information, see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges].
+
+.Drop a database and the local database alias targeting it
+====
+.Delete a database with local database aliases targeting it
+====
+The following example assumes that you have a database that is the target of a local database alias:
```
![Uploading Screenshot 2024-09-24 at 15.36.25.png…]()

Hunterness replied:
If I go to the current built version in the auto PR comment below it's still like this

renetapopova replied:
Hmm, this is odd. It doesn't look like this locally.

Hunterness replied:
that would do it XD

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

Hunterness commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,54 @@ An example could look like this:
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege.
+For more information about the privilege, see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges].
+
+.Drop a database and the local database alias targeting it
+====
+.Delete a database with local database aliases targeting it
+====
+The following example assumes that you have a database that is the target of a local database alias:
```
```suggestion
.Drop a database and the local database alias targeting it
====
The following example assumes that you have a database that is the target of a local database alias:
```

Hunterness commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,54 @@ An example could look like this:
+----
+DROP DATABASE movies CASCADE ALIASES
+----
+====
+====
```
```suggestion
====
```

Hunterness reviewed (commented):
this I think would fix the example formatting

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

renetapopova commented:
@Hunterness, any final comments?

Hunterness commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -120,23 +120,23 @@ For more information about the privilege, see xref:authentication-authorization/
 ====
-====
-The option `RESTRICT` explicitly requests the default behavior of the command.
+//The option `RESTRICT` explicitly requests the default behavior of the command.
```
Here it feels more like we should mention it somewhere, as this page have the syntax description at the top... my comment was on the composite page

renetapopova commented on modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc:
```diff
@@ -3,12 +3,63 @@
+
+This behavior is the same for the more general command `DROP DATABASE name` when using it to drop a composite database.
+
+The option `RESTRICT` explicitly requests the default behavior of the command.
```
```suggestion
//The option `RESTRICT` explicitly requests the default behavior of the command.
```

renetapopova commented on modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:
```diff
@@ -103,4 +103,54 @@ An example could look like this:
+----
+
+====
+//The option `RESTRICT` explicitly requests the default behavior of the command.
```
```suggestion
The option `RESTRICT` explicitly requests the default behavior of the command.
```

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->

neo-technology-commit-status-publisher commented:
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1805.surge.sh

Updated pages:
- [Managing database aliases in composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/aliases/manage-aliases-composite-databases)
- [Delete composite databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/composite-databases/delete-composite-databases)
- [Delete databases](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [Cypher syntax for administration commands](https://neo4j-docs-operations-1805.surge.sh/operations-manual/5/database-administration/syntax)
<!-- Sticky Pull Request Commentdocs-pr-changes -->
